js/widget.js

```diff
@@ -73,21 +73,62 @@ function gather_volume_models(model) {
  */
 function create_volume(nv, vmodel) {
   let volume = new NVImage(
-    vmodel.get("path").data.buffer,
-    volume_id(vmodel),
-    vmodel.get("colormap"),
-    vmodel.get("opacity"),
+    vmodel.get("path").data.buffer, // dataBuffer
+    volume_id(vmodel),              // name
+    vmodel.get("colormap"),         // colormap
+    vmodel.get("opacity"),          // opacity
+    undefined,                      // pairedImgData
+    vmodel.get("cal_min"),          // cal_min
+    vmodel.get("cal_max"),          // cal_max
+    undefined,                      // trustMinCalMinMax
+    undefined,                      // percentileFrac
+    undefined,                      // ignoreZeroVoxels
+    vmodel.get("visible"),          // visible
```

Comment:
I was able to put `false` here without any change of behavior. I'm not sure this property is used anywhere in niivue.